apps/mobile-wallet/locales/en-US/translation.json

```diff
@@ -10,7 +10,6 @@
   "No transactions yet": "No transactions yet",
```

Comment:
Not a bug, just not very pretty...

<img width="350" alt="image" src="https://github.com/user-attachments/assets/b327e385-a781-4f72-898f-fa625376041e">

Reply:
Seems to be fixed